subset/cloud/test_udmi

```diff
@@ -23,8 +23,9 @@ device_id=`jq -r .device_id /config/device/module_config.json`
 
 # Do basic network connectivity check
 cat /etc/resolv.conf
-ping -c 2 172.217.164.106
-ping -c 2 pubsub.googleapis.com
+ping -c 2 172.217.164.106 || true
```

Comment:
I assume this is because ping or something is blocked on GitHub actions?  If so, this is fine, but maybe add a comment explaining why it's like this.